docs/Passing-Devices-with-Clear-Containers.md

```diff
@@ -0,0 +1,66 @@
+# Using devices with Clear Containers
+
+Host devices can be passed to a container with Docker using --device option
+with docker run. An example of passing a device with docker is given below:
+```
+docker run -it --device=/dev/sdb debian
+```
+Clear Containers has support for passing vfio device groups and block
+devices to the container with --device.
+
+## VFIO Devices (PCI Passthrough)
+
+A device can be directly passed to a Clear Containers container using 
+Virtual Function I/O (VFIO). The VFIO driver is an IOMMU/device
+agnostic framework for exposing direct device access to userspace, in
+a secure, IOMMU protected environment. 
+
+For information on VFIO, please refer to:
+ - https://www.kernel.org/doc/Documentation/vfio.txt
+ - https://www.linux-kvm.org/images/b/b4/2012-forum-VFIO.pdf
+
+A device can pe passed to a Clear Container container using VFIO
+as shown below:
+
+```
+docker run -it --device=/dev/vfio/16 centos/tools bash
+```
+The user is expected to unbind the device from its host driver, bind the device
+to `vfio-pci` driver and then pass the IOMMU group that the device belongs to on
+the docker command line.
+For detailed steps on how to do this, please refer to
+https://github.com/containers/virtcontainers#how-to-pass-a-device-using-vfio-passthrough
+
+
+## Block Devices
+
+When a block device is passed with --device on the docker command line,
+Clear Containers passes the block device to the container using `virtio-block`.
+An example of passing a block device using a fake image is shown below:
+
+```
+$ # Create fake image
+$ fallocate -l 256K /tmp/test.img
+
+$ mkfs.ext4 -F /tmp/test.img
+
+$ # Find a free loop device
+$ losetup -f
+/dev/loop2
+
+$ # Use the available loop device returned by the command above
+$ sudo losetup /dev/loop2 /tmp/test.img
+
+$ # Pass the loop mounted block device to the container.
+$ docker run --device=/dev/loop2 debian stat /dev/loop2
```

Comment:
Could you uses `busybox` rather than `debian` for consistency with the other docs?